src/components/Roadmap/data/iacmData.ts

```diff
@@ -121,4 +91,44 @@ export const IacmData: Horizon = {
       },
     ],
   },
+    Released: {
+    description: "What has been released",
+    feature: [
+      {
+        tag: [{ value: "Simplify Onboarding" }],
+        title: "Customized pipeline",
+        description: "IaCM supports highly customized pipelines that allow users to create advanced workflow, add 3rd party plugins, run steps in parallel, and more.",
+      },
+      {
+        tag: [{ value: "Security" }],
+        title: "OPA integration",
+        description: "IaCM natively integrates with OPA, allowing users to define rules that can be enforced at design and runtime, eliminating policy violations and reducing the risks of security and unplanned cost",
+      },
+      {
+        tag: [],
+        title: "Cost estimation",
+        description: "Using Infracost, IaCM provides cost estimation on resource changes, allowing users to understand how cloud bill will be changed, based on resource changes",
```

Comment:
```suggestion
        description: "By utilizing Infracost, IaCM offers cost predictions for resource modifications, enabling users to foresee adjustments to their cloud expenses ahead of time.",
```

Just a suggestion, but it needs to change as the original doesn't make sense.